online.js

```diff
@@ -168,5 +169,47 @@ module.exports = {
 	},
 
 	onlineInfo: onlineInfo,
-	totalLength: totalLength
+	totalLength: totalLength,
+
+	/**
+	 * Runs the `/online` command
+	 * @param { CommandInteraction } interaction - command chat interaction
+	 * @param { string } locale - locale of the user
+	 */
+	execute: async function(interaction, locale) {
+		const onlineTags = interaction.options.getString('tag').toLowerCase().replace(/\s{2,}/g, ' ').split(' ');
```

Comment:
I didn't mean to commit this different regex, but this regex does the same thing as the original regex but slightly faster